persistence/model/src/main/java/gov/va/vro/persistence/model/AssessmentResultEntity.java

```diff
@@ -21,6 +21,8 @@ public class AssessmentResultEntity extends BaseEntity {
 
   @ManyToOne private ContentionEntity contention;
 
+  private Boolean sufficientEvidenceFlag;
```

Comment:
Please add a unit test to allow `sufficientEvidenceFlag` to be Null (when its null in the DB), as well as False. The intention of the test is to prevent this field from being changed to a primitive `boolean`.